fsspec/implementations/asyn_wrapper.py

```diff
@@ -57,8 +57,9 @@ def _wrap_all_sync_methods(self):
         """
         Wrap all synchronous methods of the underlying filesystem with asynchronous versions.
         """
+        excluded_methods = {"open"}
         for method_name in dir(self.sync_fs):
-            if method_name.startswith("_"):
+            if method_name.startswith("_") or method_name in excluded_methods:
                 continue
```

Comment:
Alternatively, this can be changed to following,

```python
for method in dir(AsyncFileSystem):
  if not method.startswith("_"):
      continue
  if not asyncio.iscoroutinefunction(getattr(AsyncFileSystem, method, None)):
      continue
  smethod = method[1:]
  f = getattr(self.sync_fs, smethod, None)
  if callable(f) and not asyncio.iscoroutinefunction(f):
      mth = async_wrapper(f, obj=self)
      setattr(self, method, mth)
      if not mth.__doc__:
          mth.__doc__ = f.__doc__
```

and maybe moved to `fsspec.asyn`.

Reply:
There is similar code in https://github.com/fsspec/filesystem_spec/blob/master/fsspec/asyn.py#L915 to differentiate methods defined in Async versus AsbtractFilesystem.